std_msgs/Header header
sdpo_driver_ros_camera/Aruco[] arucos
